htdocs/core/lib/project.lib.php

```diff
@@ -175,6 +175,17 @@ function task_prepare_head($object)
 	// $this->tabs = array('entity:+tabname:Title:@mymodule:/mymodule/mypage.php?id=__ID__');   to add new tab
 	// $this->tabs = array('entity:-tabname);   												to remove a tab
 	complete_head_from_modules($conf,$langs,$object,$head,$h,'task');
+	
+	// Manage discussion
+	if (empty($conf->global->MAIN_ALLOW_COMMENT_ON_TASK))
```

Comment:
You mean if   ! empty   I suppose

Reply:
Yes you right, and i changed the conf name.I fixed it.